readme/template-guide-readme.md

```diff
@@ -0,0 +1,101 @@
+
+## Content of the README template guide
+The following sections provide guidance and context on how to fill out the README template.
+
+### About the "Project logo and badge" section
+This section is optional. 
+
+A project logo can be helpful for users to visually identify your project. 
+
+Embed relevant badges to your README to add credibility to your project. Badges highlight important information and can help users determine project fit quickly. [Learn more about badges.](https://github.com/badges/shields)
+
+### About the "Project name" section
+The project name should be prominent in the README and easy for users to identify. You may want to consider adding the following to help identify the project:
+- Project URL
+- Name(s) of project owner(s)
+
+*Note: Consider the stage your project is at and how much information is required in your README. Names associated with the project may be better located in a separate contact document.*
+
+
+### About the "Table of contents" section
+This section is optional. 
+
+A table of contents can be useful in the case of lengthy READMEs. As the project evolves, the README may require a table of contents for information organization.
+
+*Note: For Markdown files, GitHub now automatically generates a table of contents in the file Header. [Learn more about the GitHub TOC here.](https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/)*
+
+### About the "Project description" section
+The project description is the most critical part of your README. Describe what the project does, why the reader should care about your project, and how it can help the reader. This section should be succinct, use strong verbs, and help the reader determine whether the project is of use to them.  
+
+Screenshots or videos may also be included in this section to help readers understand your project. Any assets you direct users to should exist in the repo.
+
+You can use the below project description format to start filling in your section. An example is provided.
+
+
+With *(this project)* you can *(verb)* *(noun)*...
+
+*(This project)* helps you *(verb)* *(noun)*....
+
+Unlike *(alternative)*, *)this project)* *(verb)* *(noun)*... 
+
+
+---
+
+Example: With the Chronologue project, you can discover astronomical events throughout documented history. This project helps you understand the evolution of space in a streamlined format. Unlike googling and searching for important astronomical events, the Chronologue project outlines historical events with an easy to use and simple interface.
+
+---
+
+### About the "Who is the project for" section
+Identify your project audience and state who can use the project and under what terms. 
+
+### About the "Project dependencies" section
+List any pre-requisites a user needs to interact with your project. Include relevant links to installation instructions or resources. 
```

Comment:
**Suggestion (non-blocking, phrasing):** Maybe clarify: "List any pre-requisites a user needs to interact with your project, such as dependent packages that need to be installed or technologies that are required."

A thought: do you think the users would also want an explanation of why it's a dependency? Sometimes it helps me when I'm reading stuff to know what it does so that if I'm having a problem installing a dependency I can decide if it will block my ability to use the tool or if it's just a nice to have.